lib/ui/widgets/menuButton/downloadButton.dart

```diff
@@ -93,16 +94,12 @@ class _DownloadButtonState extends State<DownloadButton> {
   }
 
   void onDownload() async {
-    var status = await Permission.storage.status;
```

Comment:
@Himanshuranjan30 could you please explain why this permission handling code was not necessary and was removed?

Reply:
Imagedownloader package automatically does that for you @LiquidatorCoder